src/styles/sfds/index.scss

```diff
@@ -146,7 +146,7 @@
     }
 
     a {
-      color: $theme-aurora-base;
+      color: var(--link);
       &:hover {
         color: $theme-aurora-lighter;
```

Comment:
Question: Would you consider updating this as part of this story or is there a later story that should take care of it?